mmf/models/fusions.py

```diff
@@ -44,7 +44,7 @@ def forward(
 
         # Case of bert encoder, we only need pooled output. For BertModelJIT encoder
         # pooled output is the 2nd in the tuple(sequence, pooled, encoded_layers)
-        if len(text) >= 2:
+        if not torch.is_tensor(text) and isinstance(text, tuple) and len(text) >= 2:
```

Comment:
Thinking more we should compare with `isinstance(text, collections.abc.Sequence)` instead of just `isinstance(text, tuple)`.

Reply:
This would cover cases where the text is a list and other iterables. I confirmed this is not true for torch.Tensor.